purchase_request_type/views/purchase_request_view.xml

```diff
@@ -18,7 +18,6 @@
                     name="request_type"
                     required="True"
                     attrs="{'readonly': [('is_editable','=', False)]}"
-                    groups="base.group_multi_company"
```

Comment:
This is not correct you should add 2 times the same field, one invisible without group, and the other with the group.

Reply:
The `request_type` field must always be shown, not only if you have the group_multi_company permission or the `purchase_request_type` module will be useless.

Reply:
Ah, sorry, I haven't seen that the field is not `company_id`.